.github/workflows/test.yaml

```diff
@@ -72,3 +72,30 @@ jobs:
           test-script: pnpm ci:test
           prnumber: ${{ steps.findPr.outputs.number }}
           base-coverage-file: coverage-reports/report.master.json
+  live-tests:
+    name: Run Live Tests
+    # if: github.head_ref = 'changeset-release/master'
+    # temp for proof of concept on GH PR
+    if: github.ref == 'refs/pull/1324/merge'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: CI Setup
+        uses: ./.github/actions/ci-setup
+
+      - name: Build
+        run: pnpm build
+
+      - name: Run Isolated Tests
+        run: pnpm test:e2e
+        env:
+          FUEL_NETWORK_URL: https://beta-4.fuel.network/graphql
+          # not enough coins to fit the target?
+          # add assets: https://faucet-beta-4.fuel.network/
+          # bech32 address: fuel1x33ajpj0jy5p2wcqqu45e32r75zrwfeh6hwqfv5un670rv4p0mns58enjg
+          # check balance: https://fuellabs.github.io/block-explorer-v2/beta-4/#/address/fuel1x33ajpj0jy5p2wcqqu45e32r75zrwfeh6hwqfv5un670rv4p0mns58enjg
```

Comment:
Would this be better placed in the live env specific test? Could have a try catch in there that logs this out?